common/annotations/en.xml

```diff
@@ -718,532 +718,548 @@ annotations.
+		<annotation cp="🙄">eyeroll | eyes | face | face with rolling eyes | rolling | whatever | shade | ugh</annotation>
 		<annotation cp="🙄" type="tts">face with rolling eyes</annotation>
-		<annotation cp="😬">face | grimace | grimacing face</annotation>
+		<annotation cp="😬">face | grimace | grimacing face | awk | awkward | dentist | grinning | smile | smiling | nothing</annotation>
 		<annotation cp="😬" type="tts">grimacing face</annotation>
-		<annotation cp="😮‍💨">exhale | face exhaling | gasp | groan | relief | whisper | whistle</annotation> <!-- 1F62E 200D 1F4A8 -->
+		<annotation cp="😮‍💨">exhale | face exhaling | gasp | groan | relief | whisper | whistle | smiley | sigh | smoke | blow | exhausted | blowing</annotation> <!-- 1F62E 200D 1F4A8 -->
 		<annotation cp="😮‍💨" type="tts">face exhaling</annotation>
-		<annotation cp="🤥">face | lie | lying face | pinocchio</annotation>
+		<annotation cp="🤥">face | lie | lying face | pinocchio | liar</annotation>
 		<annotation cp="🤥" type="tts">lying face</annotation>
 		<!-- Generated lines from Emoji Data v15, using GenerateCldrData.java -->
-		<annotation cp="🫨">earthquake | face | shaking | shock | vibrate</annotation> <!-- 1FAE8 -->
+		<annotation cp="🫨">earthquake | face | shaking | shock | vibrate | omg | panic | daze | whoa | wow | crazy | surprise</annotation> <!-- 1FAE8 -->
 		<annotation cp="🫨" type="tts">shaking face</annotation>
 		<!-- Generated lines from Emoji Data v15.1, using GenerateCldrData.java -->
 		<annotation cp="🙂‍↔">head shaking horizontally | no | shake</annotation> <!-- 1F642 200D 2194 -->
 		<annotation cp="🙂‍↔" type="tts">head shaking horizontally</annotation>
 		<annotation cp="🙂‍↕">head shaking vertically | nod | yes</annotation> <!-- 1F642 200D 2195 -->
 		<annotation cp="🙂‍↕" type="tts">head shaking vertically</annotation>
-		<annotation cp="😌">face | relieved</annotation>
+		<annotation cp="😌">face | relieved | calm | peace | relief | zen | relieved face</annotation>
 		<annotation cp="😌" type="tts">relieved face</annotation>
-		<annotation cp="😔">dejected | face | pensive</annotation>
+		<annotation cp="😔">dejected | face | pensive | awful | bored | died | losing | sucks | lost | sad | disappointed | pensive face</annotation>
 		<annotation cp="😔" type="tts">pensive face</annotation>
-		<annotation cp="😪">face | good night | sleep | sleepy face</annotation>
+		<annotation cp="😪">face | good night | sleep | sleepy face | sleeping | tired | sad | crying</annotation>
 		<annotation cp="😪" type="tts">sleepy face</annotation>
 		<annotation cp="🤤">drooling | face</annotation>
 		<annotation cp="🤤" type="tts">drooling face</annotation>
-		<annotation cp="😴">face | good night | sleep | sleeping face | ZZZ</annotation>
+		<annotation cp="😴">face | good night | sleep | sleeping face | ZZZ | bed | bedtime | nap | sleeping | tired | whatever | zzz | yawn | goodnight | night | zz | zzzz</annotation>
```

Comment:
Will the duplicate 'ZZZ' / 'zzy' be cleaned up by DAIP?

Reply:
Well, the duplicate "ZZZ" / "zzz" should be

Reply:
It should be; I'm going to run a test now.

Reply:
Yes, it does only keep one case variant. 

//ldml/annotations/annotation[@cp="😴"]

«face | good night | sleep | sleeping face | ZZZ | bed | bedtime | nap | sleeping | tired | whatever | zzz | yawn | goodnight | night | zz | zzzz»
→
«bed | bedtime | face | good night | goodnight | nap | night | sleep | sleeping | tired | whatever | yawn | zz | ZZZ | zzzz»

We still need to clean up the English to remove overlapping terms and multiwords. For this example, I'd say:

«bed | bedtime | face | good | nap | night | sleep | tired | whatever | yawn | zzz»